wordpress-mobile/WordPress-Android
# Updated gutenberg-mobile commit hash with new Story block icon

Fixes https://github.com/Automattic/stories-android/issues/618

This PR only updates the gutenberg-mobile commit hash to include changes made in https://github.com/Automattic/jetpack/pull/17962

See changes and screenshots in  https://github.com/Automattic/jetpack/pull/17962

Related Gutenberg-mobile PR https://github.com/wordpress-mobile/gutenberg-mobile/pull/2861

To test:
1. create a new gutenberg post
2. tap on the + control to show the block picker
3. scroll down and observe the Story block icon is correctly displayed now 


PR submission checklist:
